### ISSUE TITLE
Undo Interaction Does Not Work With Fenix When TalkBack On

### ISSUE THREAD

**Issue by ekager** [mozilla-mobile/android-components#8818]
The accessible delay for undo is 15000L. https://github.com/mozilla-mobile/fenix/blob/master/app/src/main/java/org/mozilla/fenix/utils/Undo.kt#L21

Can we extend the AC timeout in `UndoMiddleware` to match that? https://github.com/mozilla-mobile/android-components/blob/master/components/browser/session/src/main/java/mozilla/components/browser/session/undo/UndoMiddleware.kt#L36

You can verify this doesn't work in Fenix by turning on TalkBack, removing a tab, and waiting more than 5 seconds to undo

**Comment by ekager** [comment]
Oh I'm stupid nvm. 

This is only broken if you turn talkback while the app is running
